src/functionalTest/java/uk/gov/hmcts/reform/blobrouter/BlobRejectTest.java

```diff
@@ -42,6 +46,13 @@ void should_reject_crime_envelope_with_invalid_signature() throws Exception {
 
         // and
         assertFileInfoIsStored(fileName, config.crimeSourceContainer, REJECTED, true);
+
+        assertBlobIsPresentInStorage(
```

Comment:
we may need to wait here for a bit as well

Reply:
I think first one should enough,  if it is not in the source container it should be in target

Reply:
true!